lib/chat_api/reporting.ex

```diff
@@ -51,4 +51,30 @@ defmodule ChatApi.Reporting do
     |> select([r], %{date: fragment("date(?)", field(r, ^field)), count: count(r.id)})
     |> order_by([r], asc: fragment("date(?)", field(r, ^field)))
   end
+
+  def messages_by_weekday(account_id) do
```

Comment:
small request -- let's rename this to `count_messages_by_weekday` so that it's clear we're returning the counts and not the actual messages

Reply:
Sure, I'll update it shortly :+1: